lib/stat_tracker.rb

```diff
@@ -128,4 +129,14 @@ def fewest_tackles(season_param)
     season.fewest_tackles(season_param)
   end
 
+  def count_of_teams
+    # require 'pry'; binding.pry
+    # games_file_path = './data/games.csv'
+    # teams_file_path = './data/teams.csv'
+    # @team_collection = TeamCollection.new(teams_file_path)
+    # @game_collection = GameCollection.new(games_file_path)
+    @league_stat = LeagueStat.new(@teams_file, @games_file)
+    @league_stat.count_of_teams
+  end
+
```

Comment:
forgot to clean this up. will do that immediately as i work into the next method.